src/source-updater.js

```diff
@@ -99,23 +98,18 @@ export default class SourceUpdater {
    *      #widl-SourceBuffer-remove-void-double-start-unrestricted-double-end
    */
   remove(start, end) {
-    this.queueCallback_(function remove() {
+    this.queueCallback_(() => {
       this.sourceBuffer_.remove(start, end);
     });
   }
 
-  /**
-   * Queue an update to remove a time range from the buffer.
-   * @see http://www.w3.org/TR/media-source/
-   *      #widl-SourceBuffer-remove-void-double-start-unrestricted-double-end
-   */
   updating() {
-    return this.sourceBuffer_.updating;
+    return this.sourceBuffer_ && this.sourceBuffer_.updating;
```

Comment:
I'm wondering if this should be 

```
return !this.sourceBuffer_ || this.sourceBuffer_.updating;
```
Since code outside of sourceUpdater to determine when it is ok to load segments. Right now, the code outside will get the "all clear" when there is no `sourceBuffer_`. With the change above, outside code would think we were busy (which we kind of are; in a way) and not do anything silly quite yet.